supervisor/api/network.py

```diff
@@ -181,11 +181,23 @@ async def interface_update(self, request: web.Request) -> None:
         # Apply config
         for key, config in body.items():
             if key == ATTR_IPV4:
-                interface.ipv4 = attr.evolve(interface.ipv4, **config)
+                interface.ipv4 = attr.evolve(
+                    interface.ipv4 or IpConfig(InterfaceMethod.STATIC, [], None, []),
```

Comment:
Shouldn't we use `InterfaceMethod.AUTO` as the fallback here?

Reply:
static is the only what you assume and not send with. If you want dhcp, you send `dhcp` or if you want to disable it, you send `disabled`. It's more for CLI

Reply:
In that case, the rest look good :+1: